source/_posts/the-end-of-html-first.md

```diff
@@ -0,0 +1,18 @@
+author:
+  name: David LaPalomento
+  github: dmlap
+date: 2016-08-09 10:34:33
+tags:
+ - flash
+ - news
+ - html5 video
+---
+When video.js was [first released](https://github.com/videojs/video.js/commit/abe9252bbb7d255dd4671cf0afd93d980c2bf6df) all the way back in 2010, Flash was the only way to play video in Firefox, IE, and the Android browser. And when you *could* use HTML video, it was [really complicated to get right](http://diveinto.html5doctor.com/video.html) and broken in all sorts of scenarios (live streaming, anyone?). Those problems were a big part of why we wrote video.js in the first place. The HTML standard provided a simple, powerful, and universal API for video: why use anything else?
+
+The superiority of HTML video is pretty well established these days and [browser support for `video`](http://caniuse.com/#feat=video) reflects that. If you have your videos in MP4 format, video.js will play them natively in HTML on every modern desktop and mobile browser out there. In [video.js 5.0](http://blog.videojs.com/Video-js-5-The-Only-Thing-That%E2%80%99s-Changed-Is-Everything-except-for-like-3-things-that-didn-t-including-the-name/), we started the process of deprecating the last holdout in our supported browsers: the dreaded Internet Explorer 8. If you're saying to yourself "Wha?! You guys still support IE8??", I share your shock and horror. It's no fun but we've been holding out for the couple folks who want to use video on their sites and still have to support ancient clients. There's [a couple more hoops](http://docs.videojs.com/docs/guides/setup.html#step-1-include-the-video-js-javascript-and-css-files-in-the-head-of-your-page-) to jump through however, and you should start emotionally preparing yourself for the end of IE8 support if you're one of the people using it. With IE8 heading out to pasture, including a Flash fallback by default in video.js is starting to look a little silly.
+
+So here's what we're thinking: move Flash support out of the core of video.js and into our [legacy-compatiblity shim](https://github.com/videojs/ie8) around the time [Chrome begins deprecating Flash](https://chrome.googleblog.com/2016/08/flash-and-chrome.html) this December. We'll keep it around for awhile to support some more complex usage (say, live streaming in IE10) but the heart of video.js will go from "HTML-first" to "HTML-only." That should mean more focus from the core committers on some amazing __new__ stuff like improving our plugin framework, enhancing our support for [HLS](http://videojs.github.io/videojs-contrib-hls/) and [DASH](http://videojs.github.io/videojs-contrib-dash/), and making advanced features like [ads](https://github.com/videojs/videojs-contrib-ads) easier to integrate and better for viewers.
+
+How does that sound? Let us know in the [issues](https://github.com/videojs/video.js/issues/3520), ping [@videojs](https://twitter.com/videojs) on Twitter, or come say "hi" in the the [Slack channel](http://slack.videojs.com/).
```

Comment:
I think this would read better as: (links taken out so that I can bold and cross-out things correctly)
How does that sound? Let us know in ~~the issues~~ **this issue**, ping `@videojs` on Twitter, or come say "hi" in ~~the~~ **our** Slack channel.